RELEASE.md

```diff
@@ -50,6 +51,18 @@ Following is the Release Compatibility Matrix for PyTorch releases:
 | 1.13 | >=3.7, <=3.10 | CUDA 11.6, CUDNN 8.3.2.44 | CUDA 11.7, CUDNN 8.5.0.96 |
 | 1.12 | >=3.7, <=3.10 | CUDA 11.3, CUDNN 8.3.2.44 | CUDA 11.6, CUDNN 8.3.2.44 |
 
+## Release Cadence
+
+Following is the release cadence for year 2023/2024. Please note: these dates are subject to change, the exact dates should be announced on the [dev discuss](https://dev-discuss.pytorch.org/c/release-announcements/27) before each release.
+
+| Minor Version | Release branch cut | Release date | First patch release date | Second patch release date|
+| --- | --- | --- | --- | --- |
+| 2.1.0 | Aug 2023 | Oct 2023 | Nov 2023 | Dec 2023 |
+| 2.2.0 | Dec 2023 | Jan 2024 | Feb 2024 | Mar 2024 |
+| 2.3.0 | Mar 2024 | Apr 2024 | May 2024 | Jun 2024 |
+| 2.4.0 | May 2024 | Jul 2024 | Aug 2024 | Sep 2024 |
+| 2.5.0 | Aug 2024 | Oct 2024 | Nov 2024 | Dec 2024 |
```

Comment:
Please either change the title of first column to `Minor series` and specify versions as `2.1.X` or limit version to just major and minor)
```suggestion
| 2.1 | Aug 2023 | Oct 2023 | Nov 2023 | Dec 2023 |
| 2.2 | Dec 2023 | Jan 2024 | Feb 2024 | Mar 2024 |
| 2.3 | Mar 2024 | Apr 2024 | May 2024 | Jun 2024 |
| 2.4 | May 2024 | Jul 2024 | Aug 2024 | Sep 2024 |
| 2.5 | Aug 2024 | Oct 2024 | Nov 2024 | Dec 2024 |
```